pivot.coffee

```diff
@@ -477,6 +485,9 @@ callWithJQuery ($) ->
                 td.className = "pvtVal row#{i} col#{j}"
                 td.textContent = aggregator.format(val)
                 td.setAttribute("data-value", val)
+                #cell click callback
+                if opts.clickCallback
+                    $(td).on("click",makeClickCallback(opts.clickCallback,val,allAttrs,colKey.concat(rowKey),pivotData))
```

Comment:
so in this function we don't use jQuery, for performance reasons, as there can be *many* cells.